README.md

```diff
@@ -0,0 +1,80 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/Ruby-red)
+
+# Recipe App
+
+> 
+
+## Roadmap:
+
+✅ Created a new rails app. <br>
+✅ Followed gitflow <br>
+✅ Setup PostgreSQL database. <br>
+✅ Created a kanbanboard to track our progress <br>
+✅ The app has the following interfaces:
+
+ - A login page.
+ - A registration page.
+ - A food list (with all CRUD implementation, except for 'update').
+ - A recipes list (with all CRUD implementation, except for 'update').
+ - A general shopping list view (all missing food for all your recipes and total price).
+ - A list of all public recipes from other users with their names and total prices.
+
+
+## Built With
+
+- Ruby On Rails
+
+## To get a local copy up and running follow these simple example steps.
+
+Open your termnial and run the following command:
+
+<code>git clone https://github.com/AlexRS90/blog-app.git</code>
+ - cd recipe-app
+ - bundle
+ - rails db:create
+ - rails db: migrate
+ - <code>rails s</code> <br>
+
+ Open your browser and type the following URL <code>http://127.0.0.1:3000/</code>
+
+Finally you can start using the app and add some recipes, inventories and foods.
+Enjoy!
+
+### Testing
+
+To run the unit test once you did the above steps just write the following command in your terminal, you must be on your root directory:
+
+- <code>rspec spec</code>
+
+### Prerequisites
+
+- Ruby
+- Terminal
+
+## Authors
+
+👤 **Alejandro Ramos**
+
+- GitHub: [@AlexRS90](https://github.com/AlexRS90)
+- Twitter: [@AlejandroRBenji](https://twitter.com/AlejandroRBenji)
+- LinkedIn: [@AlexRS90](https://www.linkedin.com/in/alexrs90/)
+
+
+👤 Promise Eze<br>
+- GitHub: [@codepantherr](https://github.com/codepantherr)<br>
+- Twitter: [@codepantherr](https://twitter.com/codepantherr)<br>
+- LinkedIn: [@promise-eze](https://www.linkedin.com/in/promise-eze/)<br>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly fix the broken MIT link for a completely professional README file and repo 🙏 😄